scripts/core/datetime/reldate-directive-complex.js

```diff
@@ -36,7 +38,7 @@ function ReldateComplex(config, gettextCatalog) {
                 scope.datetimeIso = date.toISOString();
 
                 if (moment().format(COMPARE_FORMAT) === date.format(COMPARE_FORMAT)) {
-                    scope.rday = gettextCatalog.getString(date.format(DISPLAY_TODAY_FORMAT));
+                    scope.rday = gettext(date.format(DISPLAY_TODAY_FORMAT));
```

Comment:
Do not use `gettext` for dynamic values. Instead an alias of `gettext` should be used so the code inside `gettext` isn't extracted and sent to Transifex.

It doesn't look like it makes sense to translate this at all.

Reply:
Not sure if here the translation is not needed.